docs/usage.md

```diff
@@ -403,9 +401,94 @@ pvc/pvc1     Bound     pv100     20Gi       RWO                          11m
 
 NAME                          READY     STATUS    RESTARTS   AGE       IP             NODE
 po/pod1                       1/1       Running   0          11m       10.128.1.53    cld6b16
+```
+
+**Static provisioning** is a feature that is native to Kubernetes and that allows cluster admins to make existing storage devices available to a cluster. As a cluster admin, you must know the details of the storage device, its supported configurations, and mount options.
+
+To make existing storage available to a cluster user, you must manually create the storage device, a PV, and a PVC.
```

Comment:
This statement will not be completely true, as few versions need POD to be created to created volume on 3PAR. Can you confirm what are those versions in our matrix .

Reply:
I have make a generic statement to create static provisioning volume to PV,PVC and POD